notebooks/visualization/matplotlib-intermediate.ipynb

```diff
@@ -151,7 +152,7 @@
     "\n",
     "## 3. Colorizing Scatter Plots\n",
     "\n",
-    "Previously, in the [Matlibplot: Basics]() notebook, an example for drawing scatter plots was provided by setting the **linestyle** to non, and adding 'o' markers.  Another alternative is to use the **scatter** methods, while these are slower, they allow more control of the display of the data.  In this case, the data points will be colorized individually based upon a third variable."
+    "Previously, in the Matlibplot: Basics notebook, an example for drawing scatter plots was provided by setting the **linestyle** to none, and adding 'o' markers.  Another alternative is to use the **scatter** methods, while these are slower, they allow more visualization options of the data, such as style or color of the individual markers.  In this case, the data points will be colorized individually based upon a third variable."
```

Comment:
Matlibplot -> Matplotlib

Reply:
fixed the misspelling, thank you for pointing that out!